netlify-functions-ecommerce/.config/index.js

```diff
@@ -1,13 +1,11 @@
 'use strict';
 
+require('dotenv').config();
+
 if (process.env.NODE_ENV) {
-  try {
-    module.exports = require('./' + process.env.NODE_ENV);
-    console.log('Using ' + process.env.NODE_ENV); 
-  } catch (error) {
-    module.exports = require('./development');
-  }
+  console.log('Using ' + process.env.NODE_ENV + ' profile');
+  module.exports = require('./' + process.env.NODE_ENV);
 } else {
-  console.log('using production');
-  module.exports = require('./production');
+  console.log('Using jsonapi profile' + process.env.NODE_ENV);
+  module.exports = require('./jsonapi');
```

Comment:
In general, I'd recommend getting rid of this file if we're going to use dotenv. Having a multi-layered config setup that mixes different config patterns for a sample app is unnecessarily complex; makes it harder for developers to understand what is going on.